### PR TITLE
Fixes for yesterday's UI Revamp PR

### PR DESCRIPTION
**Pull request by keshav-exe**
### PR Fixes:
- 1 Hardcoded placeholder image for thumbnail
- 2 Missing packages from package.json

Resolves - No issues raised yet

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

File: src/components/CourseCard.tsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ export const CourseCard = ({
   onClick: () => void;
 }) => {
   const router = useRouter();
-  const imageUrl = 'banner_placeholder.png';
-  // course.imageUrl && course.imageUrl !== ''
-  //   ? course.imageUrl
-  //   : 'banner_placeholder.png';
+  const imageUrl =
+    course.imageUrl && course.imageUrl !== ''
```
Comment (ItsFlash10) [inline src/components/CourseCard.tsx:17]: `course.imageUrl !== ''`
it's not required as `course.imageUrl` will take care of all falsy values including an empty string

Comment (devsargam) [inline src/components/CourseCard.tsx:17]:
```suggestion
    course.imageUrl
```

**Review by devsargam — APPROVED** [review]

**Comment by devsargam** [comment]
Testing locally

File: package.json
Comment (devsargam) [inline package.json:1]: this `@radix-ui/react-separator` package is still missing

**Review by devsargam — CHANGES_REQUESTED** [review]

File: package.json
Comment (keshav-exe) [reply package.json:1]: image
it is there sir

**Pull request merged**